codegen/smithy-ruby-codegen/src/main/java/software/amazon/smithy/ruby/codegen/RubyCodeWriter.java

```diff
@@ -90,7 +90,8 @@ public RubyCodeWriter writeYardAttribute(String attribute, Runnable task) {
      */
     public RubyCodeWriter writeDocstring(String docstring) {
         doc(() -> {
-            write("$L", docstring);
+
+            writeWithNoFormatting(docstring);
```

Comment:
I think this change isn't necessary

Reply:
It's not necessary, but I find it it more clear as `writeWithNoFormatting` expresses the actual intent here.